tests/test_buffered_mode.py

```diff
@@ -72,6 +73,9 @@ def test_basic_and_nested(self):
             assert job.doc.a == 2
         assert job.doc.a == 2
 
+    @pytest.mark.xfail(
+        reason="The new SyncedCollection does not implement force_write."
```

Comment:
Can you leave a code comment indicating what to do here in the future? Like `# Remove this test in signac 2.0`? Similar for other xfail tests below.